docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -14,33 +12,35 @@ The production environment determines the methods for packaging your services, m
+* Lagom services that require an [Akka Cluster](https://doc.akka.io/docs/akka/current/cluster-usage.html) (which includes any that use the Lagom [[Persistence|PersistentEntity]] or [[Publish-Subscribe|PubSub]] APIs) must have a strategy for forming a cluster or joining an existing cluster on startup.
 
     * If you don't use a service orchestration platform and can determine the addresses of some of your nodes in advance of deploying them, Akka Cluster can be configured manually by listing the addresses of [seed nodes](https://doc.akka.io/docs/akka/current/cluster-usage.html#joining-to-seed-nodes) in the service configuration.
 
-    * Lightbend Orchestration includes open-source support for [automatic Akka Cluster formation](https://developer.lightbend.com/docs/lightbend-orchestration/current/features/akka-cluster-bootstrap.html) on Kubernetes or DC/OS.
+    * When you don't specify a no static list of `seed-nodes`, Lagom will enable the [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap.html) extension for integration with other service discovery infrastructure. Using Akka Cluster Bootstrap requires setting up a `Service Discovery` method (for example [Service Discovery for Kubernetes](https://developer.lightbend.com/docs/akka-management/current/discovery/kubernetes.html)). See the [[Lagom Cluster|Cluster]] documentation for more information.
+
+* The Akka Cluster Bootstrap feture introduced above requires enabling [Akka management HTTP](https://developer.lightbend.com/docs/akka-management/current/). That is an Akka HTTP server running on [port 8558](https://github.com/akka/akka-management/blob/v1.0.0-RC3/management/src/main/resources/reference.conf#L17) by default(that port is configurable) and shared by multiple features. Lagom will start the Akka management HTTP automatically anytime a feature that requires it is enabled. See also the [[Lagom Cluster|Cluster#Akka-Management]] documentation.
 
-    * Otherwise, you can use the open-source [Akka Cluster Bootstrap](https://developer.lightbend.com/docs/akka-management/current/bootstrap.html) extension for integration with other service discovery infrastructure, or write your own programmatic cluster formation implementation. See the [[Lagom Cluster|Cluster]] documentation for more information.
+    * Akka management HTTP also powers Akka-provided Health Checks. [Health Checks](https://developer.lightbend.com/docs/akka-management/current/healthchecks.html) is a plug-in infrastructure where library providers and users can register logic to determine if a service is alive and/or ready. Lagom will enable Akka Cluster Membership Health Checks by default any time the Lagom service requires a cluster. Akka Health Checks register, by default, the `ready/` and `alive/` routes and implement the [readiness and aliveness](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#container-probes) semantics as described by Kubernetes.
```

Comment:
Is the word "infrastructure" necessary after "plug-in"? It's easier to read without.

Reply:
How about using the same description than the actual Health checks docs?

```
Health Checks is a pluggable mechanism where library...
```